tests/conformance_dsl/context.rs

```diff
@@ -274,11 +275,11 @@ mod tests {
             .input(IonEncoding::Text)
             .expect("failed to render fragments");
 
-        let fragments_str = String::from_utf8(bytes).expect("Invalid input string generated");
-        assert_eq!(
-            fragments_str,
-            "$ion_1_1 $ion::(module _ (macro_table (macro m (v '!' ) ('%' v ) ) ) ) (:m 1)"
-                .to_string(),
-        );
```

Comment:
🪧 This test was doing a string-equality check that started failing because of the workaround I added later. I changed the test to use Ion stream equality instead.